policyengine_uk/variables/gov/dfe/extended_childcare_entitlement/extended_childcare_entitlement_eligible.py

```diff
@@ -4,10 +4,15 @@
 class extended_childcare_entitlement_eligible(Variable):
     value_type = bool
     entity = BenUnit
-    label = "Eligiblity for extended childcare entitlement"
+    label = "Eligibility for extended childcare entitlement"
     definition_period = YEAR
 
     def formula(benunit, period, parameters):
+        # Check if household is in England
+        country = benunit.household("country", period)
+        countries = country.possible_values
+        in_england = country == countries.ENGLAND
```

Comment:
Perhaps we should make this a parameter but it'd make the code ugly and we need a better system for regional parameters so leaving off.